Makefile

```diff
@@ -199,20 +199,14 @@ godocs:
 	@echo "Wait a few seconds and then visit http://localhost:6060/pkg/github.com/regen-network/regen-ledger/v3/"
 	godoc -http=:6060
 
-proto-swagger-gen:
-	@echo "Generating Protobuf Swagger"
-	@if docker ps -a --format '{{.Names}}' | grep -Eq "^${containerProtoGenSwagger}$$"; then docker start -a $(containerProtoGenSwagger); else docker run --name $(containerProtoGenSwagger) -v $(CURDIR):/workspace --workdir /workspace $(containerProtoImage) \
-		sh ./scripts/protoc-swagger-gen.sh; fi
-
-update-swagger-docs: statik
-	$(BINDIR)/statik -src=client/docs/swagger-ui -dest=client/docs -f -m
-	@if [ -n "$(git status --porcelain)" ]; then \
-		echo "\033[91mSwagger docs are out of sync!!!\033[0m";\
-		exit 1;\
-	else \
-		echo "\033[92mSwagger docs are in sync\033[0m";\
-	fi
-.PHONY: update-swagger-docs
+###############################################################################
+###                                Swagger                                  ###
+###############################################################################
+
+swagger: statik proto-update-deps
```

Comment:
Can we use docker? I'm getting this error

`
./scripts/protoc-swagger-gen.sh
unknown command "protoc" for "buf"
make: *** [Makefile:207: swagger] Error 1
`

Reply:
Are you running the script directly (i.e. `./scripts/protoc-swagger-gen.sh`) or are you using `make swagger` ? I was getting this error when running the script directly but not with `make swagger`.

Reply:
Sounds like you were running `make swagger`. Looking into it.

Reply:
I was not receiving the same error because I had an older version of `buf` installed locally.

`protoc` has been removed from `buf`. The last tendermint image version that supports `buf protoc` is `v0.3` and we are currently on `v0.7`. If we want to use the tendermint docker container with `buf protoc`, we have to use `v0.3` or we have to use `buf alpha protoc`. Using the `alpha` command for now but this may be removed in the future.

See https://github.com/bufbuild/buf/pull/915